src/newSeed/stage3.ts

```diff
@@ -49,6 +49,8 @@ export class Stage3 extends BaseStage {
       message = "Please enter a valid address for the Seed Token Address";
     } else if (!this.seedConfig.tokenDetails.maxSeedSupply || this.seedConfig.tokenDetails.maxSeedSupply === "0") {
       message = "Please enter a non-zero number for Maximum Supply";
+    } else if (this.seedConfig.seedDetails.fundingMax && this.seedConfig.seedDetails.pricePerToken && BigNumber.from(this.seedConfig.seedDetails.fundingMax).gt(BigNumber.from(this.seedConfig.tokenDetails.maxSeedSupply).mul(this.seedConfig.seedDetails.pricePerToken))) {
```

Comment:
this.seedConfig.tokenDetails.maxSeedSupply is not a BigNumber but a string that is why I had to convert to a BigNumber first and the check for the this.seedConfig.seedDetails.fundingMax and this.seedConfig.seedDetails.pricePerToken is to avoid a crash when converting to BigNumber. The engine complains when trying to convert "undefined" types to BigNumber